docs/quickstart.rst

```diff
@@ -95,7 +95,6 @@ Go to the `SR Linux` folder and then `containerlab` to spin-up the containers:
 .. code-block:: sh
 
     cd examples/Networking/SR\ Linux/containerlab
-    sudo docker pull ghcr.io/nokia/srlinux:latest
```

Comment:
https://github.com/inmanta/examples/pull/148 updates the pull policy to no longer have to explicitly pull